### ISSUE TITLE
MIDI Player improvements - Start and Stop pins/markers

### ISSUE THREAD

**Issue by BigHonkinDaddy** [powertab/powertabeditor#359]
When writing arrangements and when rehearsing sections of a score, I frequently need to repeat, over and over, a section of the score.  It would be very useful and a time saver to be able to pin a "start" and "stop" point in the score that makes it easy to have the MIDI player re-start from the start pin and play until the stop pin.  And maybe an easy way to position the MIDI player back to the start pin ready for another play/loop.

**Comment by cameronwhite** [comment]
Somewhat related to #117

**Comment by Ebdim9th** [comment]
and PT already has part of the training options already where  you can adjust the  overall percentage of the speed of playback regardless  of internal tempo changes.....

**Comment by cameronwhite** [comment]
Yep, I think the main stuff left is having a start / end marker to loop between, and maybe also options to adjust the speed gradually on each loop
